layouts/_default/single.html

```diff
@@ -40,6 +40,10 @@ <h2 class="post-title"><a href="{{ .Permalink }}">{{ .Title | markdownify }}</a>
     {{ end }}
 
     <div class="post-content">
+      {{ if .Params.Toc }}
```

Comment:
Unfortunately this won't work with global settings in `config.toml` you introduced here https://github.com/panr/hugo-theme-hello-friend/pull/141/files#diff-04c6e90faac2675aa89e2176d2eec7d8R99. `$.Site.Params.toc` is missing in the condition.

Reply:
Whoops. Fixed.